lib/fpm/package/deb.rb

```diff
@@ -905,7 +905,7 @@ def mkdir_p(dir)
   def to_s(format=nil)
     # Default format if nil
     # git_1.7.9.3-1_amd64.deb
-    return super("NAME_FULLVERSION_ARCH.TYPE") if format.nil?
+    return super("NAME_FULLVERSION_ARCH.EXTENSION") if format.nil?
```

Comment:
> I wonder why ruby people don't use the ternary if control expression?

I'm not what I would consider a "ruby" person (truthfully, I'm not quite sure what that means, so maybe I am! hehe). I have a habit of avoiding ternary operator, but I'm not sure why I do. ;P